g/GOTO++.gpp

```diff
@@ -0,0 +1,7 @@
+GOTO Le premier "Hello World"
+
+§100
+GOTOPRINTDUTEXTE() ; *`s' *`r' *`e' *`v' *`i' *`n' *`u' *` ' *`o' *`l' *`l' *`e' *`h'
```

Comment:
?

Reply:
This is the Hello World example from their examples: GOTO Le premier "Hello world"

§100
GOTOPRINTDUTEXTE() ; *`s' *`r' *`e' *`v' *`i' *`n' *`u' *` ' *`o' *`l' *`l' *`e' *`h'
§2989
GOTOPRINTDUTEXTE() ; *`d' *`l' *`r' *`o' *`w' *` ' *`o' *`l' *`l' *`e' *`h'
GOTOGOTO() %2989 ?50 %100 ?40

Reply:
fixed know, sorry for the late response i got a fever.